### PR TITLE
Add julia docker images

### PR DESCRIPTION
**Pull request by walkowif**

### CODE REVIEW

File: .github/workflows/scheduled.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: Scheduled Deployments ⏲
+
+on:
+  schedule:
+    - cron: '44 19 5,20 * *'
```
Comment (cicdguy) [inline .github/workflows/scheduled.yaml]:
```suggestion
    - cron: '44 19 10,25 * *'
```

File: scripts/install_sysdeps.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+apt-get update -y
+apt-get install -y -q ${shared_deps}
+apt-get install -y ${pkgs_to_install["${destination_image_name}"]}
+
+# Install quarto
+ARCH=$(dpkg --print-architecture)
+QUARTO_DL_URL=$(wget -qO- https://api.github.com/repos/quarto-dev/quarto-cli/releases/latest | grep -oP "(?<=\"browser_download_url\":\s\")https.*${ARCH}\.deb")
+wget -q "${QUARTO_DL_URL}" -O quarto-"${ARCH}".deb
+dpkg -i quarto-"${ARCH}".deb
+quarto check install
```
Comment (cicdguy) [inline scripts/install_sysdeps.sh]: Can we move this into a different script?

File: scripts/install_sysdeps.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+if [ "$destination_image_name" == "julia-vscode" ]; then
+  # Install VS Code server.
+  wget --no-check-certificate https://code-server.dev/install.sh -O - | sh
+  cp /init-vscode /init
+fi
```
Comment (cicdguy) [inline scripts/install_sysdeps.sh]: Also this - this isn't exactly a sysdep

**Review by cicdguy — COMMENTED** [review]
Amazing.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    SHELL=/bin/bash \
+    PATH=$PATH:/usr/local/julia
+
+WORKDIR /workspace
+
+# Copy installation scripts
+COPY --chmod=0755 [\
+    "scripts/install_sysdeps.sh", \
```
Comment (cicdguy) [inline Dockerfile:24]: Also need one script that installs additional Julia packages.

Comment (walkowif) [reply Dockerfile:24]: I added one.

File: scripts/install_julia_packages.sh [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+cat > install_pkgs.jl <<EOT
+  using Pkg;
+  pkgs=["Distributions", "CSV", "DataFrames", "JSON", "BenchmarkTools"]
+  for i in pkgs
+    Pkg.add(i)
+  end
+EOT
+fi
+
+julia install_pkgs.jl
```
Comment (cicdguy) [inline scripts/install_julia_packages.sh]: Can we make this a pure Julia script?
Change the interpreter to `#!/usr/bin/env julia` and then change the file extension to `.jl`
No need to have a shell script that creates a Julia script and then executes it.

Comment (walkowif) [reply scripts/install_julia_packages.sh]: True 😄 I'll try doing that.

Comment (walkowif) [reply scripts/install_julia_packages.sh]: Added a relevant update.

File: init-vscode [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+echo "Starting Visual Studio Code server"
+code-server \
+    --bind-addr 0.0.0.0:8081 \
+    --auth none \
+    --proxy-domain vscode \
+    --user-data-dir ~/ \
+    --extensions-dir /tmp/codeserver-extensions \
+    --disable-update-check \
+    --disable-telemetry \
```
Comment (cicdguy) [inline init-vscode]:
```suggestion
    --disable-telemetry
```

File: scripts/install_packages.jl [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    "BenchmarkTools"
+  ]
+elseif ARGS[1] == "julia"
+  pkgs=[
+    "Distributions",
+    "CSV",
+    "DataFrames",
+    "JSON",
+    "BenchmarkTools"
+  ]
```
Comment (cicdguy) [inline scripts/install_packages.jl]: @danielinteractive - Are these packages okay to start with? Anything else we need?

Comment (danielinteractive) [reply scripts/install_packages.jl]: Thanks @walkowif and @cicdguy - yeah sure, as a start those are great. Maybe can add "Plots" to the mix. And then we can always add more as needed in the specific repo workflows.

**Review by cicdguy — APPROVED** [review]
LGTM

**Pull request merged**